flask-server.py

```diff
@@ -25,122 +26,167 @@
 if not archive_dir.exists():
     archive_dir.mkdir()
 
+
 def check_auth(username, password):
     expected_credential = os.getenv("UPLOAD_CREDENTIAL")
     if expected_credential is None or expected_credential == "":
         return False
-    return username == 'loader' and password == expected_credential
+    return username == "loader" and password == expected_credential
+
 
 def login_required(f):
-    """ basic auth for api """
-    # https://stackoverflow.com/a/70317010/
```

Comment:
Why was the credit line here removed? This helps document where this code snippet came from so that any future maintainers can learn why things are the way they are